rel-eng/openshift-node.sysconfig

```diff
@@ -1,2 +1,5 @@
 ROLE="node"
 OPTIONS="--loglevel=0"
+# If running node on a separate host please rsync the admin directory ie:
+# rsync root@openshift-master:/var/lib/openshift/openshift.local.certificates/admin /var/lib/openshift/openshift.local.certificates/
+KUBECONFIG="/var/lib/openshift/openshift.local.certificates/admin/.kubeconfig"
```

Comment:
I wasn't sure if vars defined in sysconfig actually make it in as env vars given systemd's non-bash expansion of vars. If so, then yeah, much prefer defining env vars to adding cmdline options, until we have a config file.